admin/tests/integration/components/user-detail-personal-information_test.js

```diff
@@ -177,6 +177,34 @@ module('Integration | Component | user-detail-personal-information', function(ho
           // then
           assert.dom('tr[aria-label="Inscription"]').exists({ count: 2 });
         });
+
+        module('Display the schooling regestrations status', function() {
+
+          test('Should display a green tick marke on the table when "isDisabled = false"', async function(assert) {
```

Comment:
[suggestion] (petite faute)
`Should display a green tick mark on the table when "isDisabled = false"`